src/odemis/gui/comp/overlay/shapes.py

```diff
@@ -211,7 +232,13 @@ def on_char(self, evt):
         if not self.active.value:
             return super().on_char(evt)
 
-        if self._selected_shape:
+        if evt.GetKeyCode() == wx.WXK_CONTROL_Z:
+            self._undo_action = True
+            self.undo()
+        elif evt.GetKeyCode() == wx.WXK_CONTROL_Y:
```

Comment:
A lot of software (at least the I use) supports both `ctrl+y` and `ctrl+shift+z`, I find `ctrl+shift+z` easier to use, would it be easy to also support that here?

Reply:
Its not that easy
https://docs.wxpython.org/wx.KeyCode.enumeration.html#wx-keycode